sys/include/stdio_nimble.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2019 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_stdio_nimble STDIO over NimBLE
+ * @ingroup     sys
+ *
+ * @brief       Standard input/output backend using NimBLE.
+ *
+ * @{
+ * @file
+ *
+ * @author      Hendrik van Essen <hendrik.ve@fu-berlin.de>
+ */
+
+#ifndef STDIO_NIMBLE_H
+#define STDIO_NIMBLE_H
+
+#include "stdio_base.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef STDIN_BUFSIZE
```

Comment:
```suggestion
#ifndef CONFIG_STDIN_BUFSIZE
```
And same for below, configurations should be prefixed with `CONFIG`

Reply:
done, see comment below